src/idl_gen_general.cpp

```diff
@@ -99,7 +99,7 @@ const LanguageParameters& GetLangParams(IDLOptions::Language lang) {
       "",
       "",
       "",
-      "import java.nio.*;\nimport java.lang.*;\nimport java.util.*;\n"
+      "import java.nio.*;\nimport java.lang.*;\nimport java.util.*;\nimport javax.annotation.*;\n"
```

Comment:
Is this supported by every implementation of Java (and Android)? Should it be conditional upon `gen_nullable`?